### PR TITLE
ParaView 5.0.0 (update)

### PR DESCRIPTION
**Pull request by utkarshayachit**
Update to ParaView 5.0.0.

### CODE REVIEW

**Comment by dpo** [comment]
`cgns` needs to be bumped first:
```
==> brew test --verbose homebrew/science/paraview
==> FAILED
Testing homebrew/science/paraview
==> Using the sandbox
/usr/bin/sandbox-exec -f /tmp/homebrew20160110-65066-tjxp8m-0.sb /Users/brewadmin/Homebrew/Cellar/ruby187/1.8.7-p374_1/bin/ruby -W0 -I /usr/local/Library/Homebrew -- /usr/local/Library/Homebrew/test.rb /usr/local/Library/Taps/homebrew/homebrew-science/paraview.rb --verbose
==> /usr/local/Cellar/paraview/5.0.0/paraview.app/Contents/MacOS/paraview --version
dyld: Library not loaded: /usr/local/lib/libhdf5.9.dylib
  Referenced from: /usr/local/opt/cgns/lib/libcgns.3.2.dylib
  Reason: image not found
```

File: paraview.rb [hunk 1]
```diff
@@ -10,8 +11,9 @@ class Paraview < Formula
     sha256 "1a8b184f3cfa7adf51c1cb43a8cf5a6b7297cca7da3741afa0b2074e204a744a" => :mountain_lion
   end
 
-  depends_on "cmake" => :build
```
Comment (dpo) [inline paraview.rb]: You're using `cmake` on line 71.

Comment (utkarshayachit) [inline paraview.rb]: `cmak`e depedency still exists, it just moved to line 16 as I added the new option for `without-opengl2`.

Comment (dpo) [inline paraview.rb]: Apologies. I missed it. Thanks.

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by utkarshayachit** [comment]
Please let me know if there's anything I can do to get this merged. If I am not mistaken, looks like the testbot keeps on failing due to timeouts. That's not too surprising. ParaView can take a long time to build on slower systems.

**Comment by dpo** [comment]
@mikemcquaid This is another one where we need more than 60 minutes to build. Any chance to get such resources at some point?

**Comment by mikemcquaid** [comment]
@dpo I'll do it when I get the chance. I think we'll need to work out a better long-term solution for this, though.

**Comment by mikemcquaid** [comment]
CC @DomT4 as you often keep a closer look at the build queue than I do now.

**Comment by DomT4** [comment]
https://github.com/Homebrew/homebrew-science/pull/3314#issuecomment-188310049

**Comment by dpo** [comment]
This just won't build under the time constraints so I'd like to ask a question here. Is there any advantage to having this formula in Homebrew/Science given that it's available from Homebrew/Cask?

@utkarshayachit @davydden

**Comment by utkarshayachit** [comment]
@dpo The one reason for having this formula in homebrew-science is that it will enable SDK installs in near future (currently it doesn't do that). That would be needed for folks trying to develop plugins for ParaView or custom applications or instrumenting simulation codes to use ParaView insitu components.

**Comment by davydden** [comment]
I, personally, do not build paraview and rather use the homebrew-cask version. 

As far as i recall, building paraview could be advantageous when one wants to add some plugins which are not included automatically. So i would keep it.

**Comment by dpo** [comment]
Ok. Thanks both for the feedback. That makes sense.

**Comment by DomT4** [comment]
Don't rely on Travis here.

If you need bottles you'll only have to go through Jenkins regardless to get them. Just wait till 23:00 - 09:00 UK time (18:00 - 04:00 EST), check the CI queue is clear/clearish, set the Science timeout north of 4 hours and re-queue the build by calling `@BrewTestBot` with the magical phrase.

As long as you revert the timeout relatively quickly after the build completes it's fine to go higher temporarily.

**Comment by dpo** [comment]
If I have authority to do that, I don't know how.

**Comment by DomT4** [comment]
Fairly sure if you can access the bot you're allowed to play with the timeouts. Can you access the http://bot.brew.sh/job/Homebrew%20Science%20Pull%20Requests/configure page?

**Comment by dpo** [comment]
Ok thanks.

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
This one failed to build on all three platforms, sadly. Actual build failure rather than timeout, though.

**Comment by dpo** [comment]
Sorry about that. I'll check it out locally.

**Comment by dpo** [comment]
The bug is upstream and related to recent changes in `ffmpeg`, similar to #3380.

**Comment by utkarshayachit** [comment]
Thanks @dpo, I will push a fix to upstream and this PR to address this issue.

**Comment by dpo** [comment]
It seems 5.1 is out now. @utkarshayachit do you want to push an update?

**Comment by utkarshayachit** [comment]
@dpo Yes, I'll have an update soon. Should I close this PR and open a new one when that happens, or just update this one?

**Comment by dpo** [comment]
Let's open a new one. I'll close this.

**Pull request closed without merging**